docs/docs/guides/How to Add Digest to In-App Notifications.md

```diff
@@ -0,0 +1,293 @@
+
+const Header = () => {
+  function onNotificationClick(message) {
+    // your logic to handle the notification click
+    if (message?.cta?.data?.url) {
+      window.location.href = message.cta.data.url;
+    }
+  }
+  return (
+    <div className="bell">
+      <div className="intro">
+        <h1 className="title">Play with the digest engine!</h1>
+        <p className="desc">
+          Don't know how to? Start <a href="https://docs.novu.co/platform/digest/">here</a>.
+        </p>
+      </div>
+      <div className="center">
+        <NovuProvider
+          subscriberId={'aa234u787'}
+          applicationIdentifier={process.env.REACT_APP_YOUR_APP_ID_FROM_ADMIN_PANEL}
+        >
+          <PopoverNotificationCenter onNotificationClick={onNotificationClick}>
+            {({ unseenCount }) => <NotificationBell unseenCount={unseenCount} />}
+          </PopoverNotificationCenter>
+        </NovuProvider>
+      </div>
+    </div>
+  );
+};
+
+export default Header;
+```
+
+You’ll also need to plug in your `applicationIdentifier`. You can get your `application id` from the settings menu in the Novu dashboard.
+
+Beyond this, we just need to call the API we had written above when the form is submitted with the data we want in the payload. The code to do so is as follows:
+
+```jsx
+import { useState } from 'react';
+import axios from 'axios';
+import '../css/Body.css';
+
+const Body = () => {
+  const [name, setName] = useState('');
+
+  const onSubmitHandler = async (e) => {
+    e.preventDefault();
+    const res = await axios.post('http://localhost:3000/api/v1/sending-digest', { name });
+    setName('');
+  };
+  const onChangeHandler = (e) => {
+    setName(e.target.value);
+  };
+  return (
+    <form onSubmit={onSubmitHandler} className="form">
+      <label for="noti-text" className="label">
+        Enter notification text:
+      </label>
+      <input
+        id="noti-text"
+        placeholder="Will be sent as a digest message"
+        type="text"
+        onChange={onChangeHandler}
+        value={name}
+        className="text"
+      />
+      <button className="btn" type="submit">
+        {' '}
+        Send{' '}
+      </button>
+    </form>
+  );
+};
+
+export default Body;
+```
+
+The app is done now!
+
+Congratulations if you’ve followed the guide up until here. We can style our app a little using [some CSS](https://github.com/novuhq/digest-learning-app/blob/main/frontend/src/App.css).
+
+If you’ve done everything as recommended, you’ll end up with an app that looks like this:
+
+![out.gif](https://res.cloudinary.com/dxc6bnman/image/upload/v1688130966/guides/out_ca2bjr.gif)
+
+In this app, when we enter some text in the input box and click send, the notification appears in the bell icon after the delay specified above (when we’re creating the workflow):
+
+You can find the entire code(frontend as well as backend) for this app [here](https://github.com/novuhq/digest-learning-app/tree/main).
```

Comment:
Let's move this part to the top of the guide. Just below the first paragraph in the Introduction section.